Frameworks/UIKit/UIApplicationMain.mm

```diff
@@ -320,6 +320,14 @@ void _UIApplicationShutdown() {
     [[UIApplication sharedApplication] _sendHighMemoryWarning];
 }
 
+extern "C" void UIApplicationMainHandleSuspendEvent() {
+    [[UIApplication sharedApplication] _sendSuspendEvent];
```

Comment:
>_sendSuspendEvent [](start = 39, length = 17)

nit: we are not sending any suspend event (unlike other methods) we are handling it.